packages/api/src/platforms/vtex/resolvers/query.ts

```diff
@@ -168,4 +169,21 @@ export const Query = {
       familyName: profile?.lastName?.value ?? '',
     }
   },
+  region: async (
+    _: any,
+    {
+      input,
+    }: {
+      input: RegionInput
+    },
```

Comment:
I know it's just one variable, but how about creating a type for it? Like `QueryRegionArgs`, to keep the same pattern as `QueryProductArgs` for example

Reply:
Totally agree with.